docs/3rd_party/pdf/snippets/fpdf2/create_invoice.py

```diff
@@ -0,0 +1,42 @@
+from fpdf import FPDF
+
+
+def create_custom_invoice(title, subtitle, items, filename="invoice.pdf"):
+    """
+    Creates a custom invoice PDF document using fpdf2.
+
+    Args:
+    - title: Title of the invoice.
+    - subtitle: Subtitle of the invoice.
+    - items: List of tuples containing item details (name, quantity, price).
+    - filename: Name of the output PDF file.
+
+    Example:
+    >>> create_custom_invoice(
+    ...     "Invoice",
+    ...     "Number #123",
+    ...     [
+    ...         ("Software Development 1", 1, 5500),
+    ...         ("Consultancy 2", 1, 1000),
+    ...         ("Equipment", 3, 300),
+    ...     ]
+    ... )
+    """
+    pdf = FPDF()
+
+    pdf.add_page()
+    pdf.set_font("Arial", size=12)
+
+    pdf.cell(200, 10, txt=title, ln=True, align="C")
+    pdf.cell(200, 10, txt=subtitle, ln=True, align="C")
```

Comment:
Would help the reader to understand what the magic `"C"` means through a comment or constant declaration.